contrib/dracut/90zfs/zfs-lib.sh.in

```diff
@@ -74,14 +74,22 @@ import_pool() {
 # mount_dataset DATASET
 #   mounts the given zfs dataset.
 mount_dataset() {
-        dataset="${1}"
+    dataset="${1}"
     mountpoint="$(zfs get -H -o value mountpoint "${dataset}")"
-
+    mountopts="strictatime"
+    ATIME_OPTION=$(zfs get -H -o value -s default,local,inherited atime ${dataset})
+    if [ "${ATIME_OPTION}" = "off" ]; then
+        mountopts="noatime"
+    fi
```

Comment:
The `RELATIME_OPTION` section should be in an `else` block off of `if [ "${ATIME_OPTION}" = "off" ];`

Reply:
despite claims in the manpage that relatime only works with atime=on, that is not the case. it works either with atime=on or off, and we should fix the man page instead.

Reply:
Got it